### PR TITLE
Detailed Report Messages for Google Calendar

### LINKED ISSUE #173: Detailed Report Message - Calendar
This issue wasn't copied over.

@LaurenBassett The RNG chose you to copy over Calendar's detailed reports Rego code over to this repo.

### PR DESCRIPTION
**Pull request by LaurenBassett**
## 🗣 Description ##

Adds detailed report messages to calendar, and removes any detailed report messages that use other functions. 
### 💭 Motivation and context

Closes #173 

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] If applicable, *All* future TODOs are captured in issues, which are referenced in the PR description.
- [x] The relevant issues PR resolves are linked preferably via [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).
- [x] All relevant type-of-change labels have been added.
- [x] I have read and agree to the [CONTRIBUTING.md](https://github.com/cisagov/ScubaGoggles/blob/main/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge Checklist

- [ ] This PR has been smoke tested to ensure main is in a functional state when this PR is merged.
- [ ] Squash all commits into one PR level commit using the `Squash and merge` button.

## ✅ Post-merge Checklist

- [ ] Delete the branch to clean up.
- [ ] Close issues resolved by this PR if the [closing keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) did not activate.

### CODE REVIEW

File: rego/Calendar.rego [hunk 1]
```diff
@@ -286,7 +249,10 @@ tests contains {
 # GWS.CALENDAR.4 #
 ##################
 
-NonCompliantOUs4_1 contains OU if {
+NonCompliantOUs4_1 contains {
+    "Name": OU,
+    "Value": concat("", ["Paid calendar appointments are enabled."])
```
Comment (adhilto) [inline rego/Calendar.rego:254]: Nothing to concat here, recommend simplifyling
```suggestion
    "Value": "Paid calendar appointments are enabled."
```

**Review by adhilto — APPROVED** [review]
Overall looks good. I have one minor style comment, but I'll approve either way.

File: rego/Calendar.rego [hunk 2]
```diff
@@ -87,40 +87,14 @@ if {
...
-        " and outsiders can manage calendars"
-    ])
-}
+GetFriendlyValue1_2(Value) := "Secondary Calendars Share all information, but outsiders cannot change calendars." if {
+    Value == "READ_ONLY_ACCESS"
+} else := "Share all information, and outsiders can change calendars." if {
+    Value == "READ_WRITE_ACCESS"
+} else := "Share all information, and allow managing of calendars." if {
+    Value == "MANAGE_ACCESS"
+} else := "Only free/busy information (hide event details)."
```
Comment (mitchelbaker-cisa) [inline rego/Calendar.rego:97]: Suggest adding a condition to handle the "SHOW_ONLY_FREE_BUSY_INFORMATION" setting name so we handle all cases, then have the last else default to the `Value` parameter. 

Example from chat: 

```
#
# Baseline GWS.CHAT.2.1v0.1
#--
GetFriendlyValue2_1(Value) := "Allow all files" if {
    Value == "ALL_FILES"
} else := "Images only" if {
    Value == "IMAGES_ONLY"
} else := Value
```

**Review by mitchelbaker-cisa — COMMENTED** [review]
Left one comment for your review. Other than that everything looks good and all unit tests pass.

File: rego/Calendar.rego (hunk 2 above)
Comment (LaurenBassett) [reply rego/Calendar.rego]: Just pushed those changes through

**Review by mitchelbaker-cisa — APPROVED** [review]

**Pull request merged**